src/sentry/tasks/post_process.py

```diff
@@ -279,6 +295,14 @@ def process_snoozes(group):
         return False
 
     if not snooze.is_valid(group, test_rates=True):
+        snooze_details = {
+            "until": snooze.until,
+            "count": snooze.count,
+            "window": snooze.window,
+            "user_count": snooze.user_count,
+            "user_window": snooze.user_window,
+        }
```

Comment:
Still want to see a schema for this.

Reply:
Schema: Coming to a PR near you before internal